src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const values = document.querySelectorAll('.population');
+const avarage = document.querySelector('.average-population');
+const total = document.querySelector('.total-population');
+let totl = 0;
```

Comment:
The variable name 'totl' is misspelled. It should be 'total' to match the intended purpose and improve code readability.